app/src/routes/return.tsx

```diff
@@ -1,125 +1,208 @@
+        <Button
+          disabled={selectedCondition === "" ? true : false}
+          variant="secondary"
+          onClick={submitCondition}
+        >
+          Submit
+        </Button>
+      </Modal.Footer>
+    </Modal>
+  ) : null;
+};
 
 const Notification = ({ show, type, onClick, id }) => {
-    return show ? (
-        <div className="position-absolute bottom-0 notif">
-            <div className="graphic"/>
-            <div className="text-group">
-                <div style={{fontSize: "14px", fontWeight: 400}}>Successfully {type}</div>
-                <div style={{fontSize: "16px", fontWeight: 600}}>Mug ID # {id}</div>
-            </div>
-            <Button className="report-btn" variant="secondary" onClick={onClick}>
-                <div className="octagon">
-                    <FontAwesomeIcon icon={faExclamation} size="xs" transform="up-2"/>
-                </div>
-            </Button>
+  return show ? (
+    <div className="position-absolute bottom-0 notif">
+      <div className="graphic" />
+      <div className="text-group">
+        <div style={{ fontSize: "14px", fontWeight: 400 }}>
+          Successfully {type}
+        </div>
+        <div style={{ fontSize: "16px", fontWeight: 600 }}>Mug ID # {id}</div>
+      </div>
+      <Button className="report-btn" variant="secondary" onClick={onClick}>
+        <div className="octagon">
+          <FontAwesomeIcon icon={faExclamation} size="xs" transform="up-2" />
         </div>
-    ) : null;
-}
+      </Button>
+    </div>
+  ) : null;
+};
 
 export default () => {
-    const [scanId, setScanId] = useState("")
-    const [showNotif, setShowNotif] = useState(false);
-    const [popUp, setPopUp] = useState(false);
-    const [notifType, setNotifType] = useState("returned");
+  const [scanId, setScanId] = useState("");
+  const [showNotif, setShowNotif] = useState(false);
+  const [popUp, setPopUp] = useState(false);
+  const [notifType, setNotifType] = useState("returned");
 
-    const navigate = useNavigate()
+  const { sessionToken } = useAuth();
+  const navigate = useNavigate();
 
-    const onScan = async (id: string) => {
-        setScanId(id);
-        let val = await DishAPI.updateDishReturn(id);
-        // if an animation is added, this will re-trigger it
-        if (!(val === null)) {
-            if (showNotif)
-                setShowNotif(false);
-            setNotifType("returned");
-            setShowNotif(true);
+  const onScan = async (id: string) => {
+    setScanId(id);
+    try {
+      let response = await axios.post(
+        `${config.serverUrl}/api/dish/return?qid=${id}`,
+        {
+          returned: {
+            broken: false,
+            lost: false,
+          },
+        },
+        {
+          headers: {
+            "x-api-key": config.apiKey,
+            "session-cookie": sessionToken,
+          },
         }
+      );
+
+      if (response && response.status === 200) {
```

Comment:
you can actually use `if (res.ok)` here, which is true if the response is 2xx and false otherwise

Reply:
Thanks! I checked, but seems like axios doesn't have this, only fetch does.